docs/getting-started/security-ui.asciidoc

```diff
@@ -91,6 +91,14 @@ The Alerts page allows you to view and manage all alerts to monitor activity wit
 [role="screenshot"]
 image::detections/images/alert-page.png[]
 
+[float]
+=== Findings page
+
+The Findings page provides an overview of your Kubernetes infrastructure configuration's performance on a range of security benchmarks. Refer to <<kspm, Kubernetes security posture management>>.
+
+[role="screenshot"]
+image::cloud-native-security/images/findings-page.png[The Findings page]
```

Comment:
This image looks a little fuzzy on my monitor -- usually that happens if the image was scaled down when it was saved. Snagit has an option to do this, but I always make sure it's unchecked to get a higher-quality image:
<img width="500" alt="image" src="https://user-images.githubusercontent.com/87339667/185968761-4e1e7e61-c601-45c3-990f-7034762b6ca0.png">